docs/architecture.rst

```diff
@@ -264,3 +276,35 @@ output:
    :alt: Keeper state machine
 
    Keeper State Machine
+
+
+Multi-nodes architecture
```

Comment:
I guess `Multi-node` is slightly misleading as even single standby case we have `multi-node`. So, maybe `Multiple-standby architecture` fits better?

Reply:
Fixed now.